content/blog/2017-09-04-promcon-2017-recap.md

```diff
@@ -0,0 +1,122 @@
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">Cloudflare is a large user of Prometheus. Lorenz Bauer explains how they handle alerts and dashboards across 115 data centers <a href="https://twitter.com/hashtag/PromCon2017?src=hash">#PromCon2017</a> <a href="https://t.co/IXs09Lp6z6">pic.twitter.com/IXs09Lp6z6</a></p>&mdash; PrometheusMonitoring (@PrometheusIO) <a href="https://twitter.com/PrometheusIO/status/898461326191861760">August 18, 2017</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">DigitalOcean uses Prometheus to monitor <a href="https://twitter.com/kubernetesio">@kubernetesio</a> deployed services. <a href="https://twitter.com/snehainguva">@snehainguva</a> talks about their setup and pitfalls <a href="https://twitter.com/hashtag/PromCon2017?src=hash">#PromCon2017</a> <a href="https://t.co/dPaSIMybjO">pic.twitter.com/dPaSIMybjO</a></p>&mdash; PrometheusMonitoring (@PrometheusIO) <a href="https://twitter.com/PrometheusIO/status/898455013281984512">August 18, 2017</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+Speakers from [Grafana](https://grafana.com/) and [InfluxData](https://www.influxdata.com/) brought us up to date with new features and Prometheus integrations:
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">Next, <a href="https://twitter.com/CarlBergquist">@CarlBergquist</a> from <a href="https://twitter.com/raintanksaas">@raintanksaas</a> shows cool tricks and new features in <a href="https://twitter.com/grafana">@grafana</a>. <a href="https://twitter.com/hashtag/PromCon2017?src=hash">#PromCon2017</a> <a href="https://t.co/aXCbOAA1dp">pic.twitter.com/aXCbOAA1dp</a></p>&mdash; PrometheusMonitoring (@PrometheusIO) <a href="https://twitter.com/PrometheusIO/status/898119125540765697">August 17, 2017</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">You like Prometheus but have a need for long-term storage? <a href="https://twitter.com/pauldix">@pauldix</a> presents how <a href="https://twitter.com/InfluxDB">@InfluxDB</a> integrates with Prometheus at <a href="https://twitter.com/hashtag/PromCon2017?src=hash">#PromCon2017</a> <a href="https://t.co/hpsO95mPVL">pic.twitter.com/hpsO95mPVL</a></p>&mdash; PrometheusMonitoring (@PrometheusIO) <a href="https://twitter.com/PrometheusIO/status/898476044327497728">August 18, 2017</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+Several Prometheus core developers also spoke about best practices and new features and developments in Prometheus:
```

Comment:
The social aspects and staleness talks should go in here too.

Reply:
I was worried about this section getting too long, but happy to make it longer if you think that's better...